packages/blade/src/tokens/theme/overrideTheme.ts

```diff
@@ -0,0 +1,24 @@
+import { cloneDeep, isEqual, merge, isPartialMatchObjectKeys, DeepPartial } from '../../utils';
+import { paymentTheme, bankingTheme } from '../theme';
+import type { Theme } from '../theme';
+
+type OverrideTheme = {
+  baseTheme: Theme;
+  overrides: DeepPartial<Theme>;
```

Comment:
How about calling this `themeOverrides`?

Reply:
seems redundant overrideTheme is the func name then again themeOverrides